client/my-sites/plans/current-plan/trials/use-business-trial-included-features.ts

```diff
@@ -82,8 +85,7 @@ export default function useBusinessTrialIncludedFeatures(
 			illustration: seoTools,
 			showButton: true,
 			buttonText: translate( 'Increase visibility' ),
-			buttonClick: () =>
-				( location.href = localizeUrl( 'https://wordpress.com/support/seo-tools/' ) ),
+			buttonClick: () => page.redirect( `/marketing/traffic/${ siteSlug }` ),
```

Comment:
I'm suggesting calling redirection logic by invoking `page( URL )`, since the `.redirect` method doesn't create history. If you trigger the browser Back action, you will not be redirected to the `my-plan` page because it's replaced with `marketing/traffic`.